packages/vite-plugin/src/main.ts

```diff
@@ -69,7 +69,6 @@ export default function netlify(options: NetlifyPluginOptions = {}): any {
 
           next()
         })
-        logger.log(`Middleware loaded. Emulating features: ${netlifyDev.getEnabledFeatures().join(', ')}.`)
```

Comment:
I don't see the value of this. It won't mean anything to anyone using the Vite plugin through a meta-framework.

Reply:
https://github.com/netlify/primitives/pull/308/commits/673d4e575078c83c5cf61889f7faba26b78f847e